src/Model/Term.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * @file src/Model/Term
+ */
+namespace Friendica\Model;
+
+use dba;
```

Comment:
Please add `require_once 'include/dba.php';` until the `dba` class is moved to `src`.